### ISSUE TITLE
Collect blips from podsie

### ISSUE THREAD

**Issue by jorge-vasquez-acn** [AlleyCorpNord/acn-radar#59]
Book podsie team for brainstorming session and collect information into blips.

**Comment by opiation** [comment]
Hey @jorge-vasquez-acn, I keep seeing this issue on my list but I don't know what the acceptance criteria are or how to advance on it.

Care to book @sometechchick and me for a morning chat some time between Monday and Thursday inclusively next week or the week after?

**Comment by jorge-vasquez-acn** [comment]
Yes @opiation , sorry I think I wanted to do this a few weeks ago but both you and @sometechchick  were off so will book something for next week with both of you!

**Comment by opiation** [comment]
Hey @sometechchick, I'm thinking the following 2 tools to the tech radar that we're using with Podsie for now:
- Sorbet (the static/runtime type system built by Stripe) for its LSP refactoring advantages and verbose type signature pain points/limitations
- Vite, the JS/TS bundling / dev server tooling used in the `podsie-teacher` front-end as a relatively easy migration path away from `create-react-app`

I'll add them to this issue description.

**Comment by jorge-vasquez-acn** [comment]
@opiation:  Any progress with this?  Is there any way that I can help?
